jfrog-applications/jfrog-cli/cli-command-summaries.md

```diff
@@ -0,0 +1,117 @@
+# CLI Command Summaries
+
+## Overview
+
+The command summaries feature allows the recording of JFrog CLI command outputs into the local file system. This can be used to display a summary of the executed CLI command.
```

Comment:
Let's explain here why is the "recording" (aggregation) needed. We should explain that it is used to allow showing a summary in the scope of entire pipeline (a sequence of commands that represent a workflow).